ivadomed/loader/utils.py

```diff
@@ -387,6 +387,5 @@ def update_metadata(metadata_src_lst, metadata_dest_lst):
             metadata_dest_lst = metadata_dest_lst + [SampleMetadata({})] * \
                                 (len(metadata_src_lst) - len(metadata_dest_lst))
         for idx in range(len(metadata_src_lst)):
-            for i_dest in range(len(metadata_dest_lst)):
-                metadata_dest_lst[i_dest]._update(metadata_src_lst[idx], TRANSFORM_PARAMS)
+            metadata_dest_lst[idx]._update(metadata_src_lst[idx], TRANSFORM_PARAMS)
```

Comment:
With the change in the transform decorator --> we would need only to update metadata_src_lst[0] with metadata_dest_lst[0] :-)

Reply:
Changed it :)